web/gui/goto-host-from-alarm.html

```diff
@@ -92,6 +93,9 @@
 }
 
 function netdataURL(url) {
+    if(urlOptions.alarm_when != 0) {
+        urlOptions.alarm_when -= 150;
```

Comment:
Why -150? What does that do?

Reply:
In normal situation our charts has 5 minutes(300 seconds), so here I was doing a test where I was trying to centralize the document in the 300 second that I have in my screen, but of course this is not good, because case some user try to see an alarm that happens in the first second he won't have data centralized. 
When I wrote this part of the code, I did not have enough knowledge of our java script files, now that I got the necessary knowledge, I will remove this line for the next commit, but it was useful in the beginning for I understand the whole process.